src/View/Form/EntityContext.php

```diff
@@ -426,6 +426,29 @@ public function isRequired($field)
         return false;
     }
 
+    /**
+     * Get field length from validation
+     *
+     * @param string $field The dot separated path to the field you want to check.
+     * @return int|null
+     */
+    public function getMaxLength($field)
+    {
+        $parts = explode('.', $field);
+        $validator = $this->_getValidator($parts);
+        $fieldName = array_pop($parts);
+        if (!$validator->hasField($fieldName)) {
+            return null;
+        }
+        foreach ($validator->field($fieldName)->rules() as $rule) {
+            if ($rule->get('rule') === 'maxLength') {
+                return $rule->get('pass')[0] ?? null;
```

Comment:
Null coleasing operator `??` is not supported in PHP 5.6.

Reply:
Basic isset() ternary suffices.

Reply:
> Null coleasing operator `??` is not supported in PHP 5.6.

Would you like support 5.6? I can change it if you would.